src/GNNGraphs/GNNGraphs.jl

```diff
@@ -16,6 +16,7 @@ using LinearAlgebra, Random, Statistics
 import MLUtils
 using MLUtils: getobs, numobs, ones_like, zeros_like
 import Functors
+using SimpleWeightedGraphs
```

Comment:
We don't want to carry an extra dependency, so this should be part of an [extension](https://pkgdocs.julialang.org/v1/creating-packages/#Conditional-loading-of-code-in-packages-(Extensions)).

Also, the compact entry is missing.